test/test_functional.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import torch
+import torchaudio
+import unittest
+import test.common_utils
+
+
+class TestFunctional(unittest.TestCase):
+    # size (2,20)
+    test_data = torch.tensor([
```

Comment:
Instead of using a static size, I'd suggest you create a function with a few parameters that can be used to deterministically (hence repeatably) create Tensors of various properties (sizes, dtypes, ranges, etc.).